loki/transformations/single_column/annotate.py

```diff
@@ -221,14 +266,16 @@ def device_alloc_column_locals(cls, routine, column_locals):
             routine.body.prepend((ir.Comment(''), pragma, ir.Comment('')))
```

Comment:
[no action required] This is not covered in tests, so we don't seem to be testing this.
(@mlange05 for info)